src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+import java.io.IOException;
+
+/**
+ * The <code>ExitCommand</code> is created when the user enters <code>"bye"</code>. The exit command will show the exit
+ * message and save all tasks in the list of tasks in the {@link TaskList} class object to the
+ * {@link duke.storage.Storage}. The storage is the file path specified by {@link duke.main.Duke} and
+ * {@link duke.storage.Storage}.
+ */
+public class ExitCommand implements Command {
+
+    /**
+     * Executes the command. This will save the list of tasks in {@link TaskList} into the storage. The storage is the
+     * file path specified by {@link duke.main.Duke} and {@link Storage}.
+     * @param tasks the list of tasks
+     * @param ui the user interface
+     * @param storage the storage for the tasks
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        try {
+            storage.save(tasks.save());
```

Comment:
Good idea to only save the tasks after all commands have been ran, rather than saving and updating time after time.